README.md

```diff
@@ -50,13 +50,13 @@ Participation in the Zcash project is subject to a
 
 ### Building
 
-Build Zcash along with most dependencies from source by running:
+Build Zcash along with most dependencies from source by running the following command:
 
 ```
 ./zcutil/build.sh -j$(nproc)
 ```
 
-Currently only Linux is officially supported.
+Currently, Zcash is only built for Linux.
```

Comment:
I think the original is more accurate for this line as it builds on multiple platforms/architectures but only Linux is officially supported?

Reply:
Sounds good! Ty!

Reply:
I agree. Please make this change, or something along the lines of:

```suggestion
Currently, Zcash is only officially supported on Debian and Ubuntu.
```

(swapping out `Linux` for [greater accuracy](https://ci.z.cash/#/builders?tags=%2Bsupported)).